tasks/53_diff-array/diff-array.js

```diff
@@ -0,0 +1,26 @@
+export const diffArray = (arr1, arr2) => {
+  for (let i = arr1.length - 1; i >= 0; i--) {
```

Comment:
Followup - is there any other solution with better complexity than O(n^2) ? Maybe something with hashmap?